libsolidity/analysis/ControlFlowAnalyzer.cpp

```diff
@@ -110,8 +110,12 @@ void ControlFlowAnalyzer::checkUninitializedAccess(CFGNode const* _entry, CFGNod
 
 		// Propagate changes to all exits and queue them for traversal, if needed.
 		for (auto const& exit: currentNode->exits)
-			if (nodeInfos[exit].propagateFrom(nodeInfo))
+		{
+			bool exists = nodeInfos.find(exit) != nodeInfos.end();
+
+			if (nodeInfos[exit].propagateFrom(nodeInfo) || !exists)
 				nodesToTraverse.insert(exit);
+		}
```

Comment:
If desired, this could be shortened to e.g.:
```suggestion
			if (
				auto exitInfo = valueOrNullptr(nodeInfos, exit);
				!exitInfo || exitInfo->propagateFrom(nodeInfo)
			)
				nodesToTraverse.insert(exit)
```
but also fine as is.

Reply:
Actually, nevermind this. It needs to create the entry right here and already propagate from the entry.

Reply:
I.e. the suggestion was wrong, resolving.

Reply:
Aaaah, the new c++ things we can do nowadays

Reply:
Though I think we want to call propagateFrom in every case, so I am reverting your `if` conditions.